cmd/disconnect.go

```diff
@@ -0,0 +1,32 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/railwayapp/cli/entity"
+	"github.com/railwayapp/cli/ui"
+)
+
+func (h *Handler) disconnectWithId(ctx context.Context, req *entity.CommandRequest) error {
+	return nil
+}
```

Comment:
This doesn't seem used anywhere so can be removed

Reply:
oops^2